eva-accession-core/src/test/java/uk/ac/ebi/eva/accession/core/SubmittedVariantAccessioningServiceTest.java

```diff
@@ -218,6 +218,41 @@ public void getOrCreateAccessionsInBothRepositories() throws AccessionCouldNotBe
         assertEquals(3, submittedVariants.size());
     }
 
+    @UsingDataSet(locations = {"/test-data/dbsnpSubmittedVariantEntity.json"})
+    @Test
+    public void getOrCreateDbsnpIdenticalVariants() throws AccessionCouldNotBeGeneratedException {
+        SubmittedVariant variantPresentInDbsnp = new SubmittedVariant("GCA_000009999.3", 9999, PROJECT_DBSNP, "21",
+                                                                      20849999, "", "GG", null);
+
+        List<AccessionWrapper<ISubmittedVariant, String, Long>> generatedAccessions = service.getOrCreate(
+                Collections.singletonList(variantPresentInDbsnp));
+        assertEquals(1, generatedAccessions.size());
+    }
+
+    @UsingDataSet(locations = {"/test-data/dbsnpSubmittedVariantEntity.json"})
+    @Test
+    public void getOrCreateDbsnpEquivalentVariants() throws AccessionCouldNotBeGeneratedException {
+        SubmittedVariant identicalVariantPresentInDbsnp = new SubmittedVariant("GCA_000009999.3", 9999, PROJECT_DBSNP,
+                                                                               "21", 20849999, "", "GG", null);
+
+        List<AccessionWrapper<ISubmittedVariant, String, Long>> generatedAccessions = service.getOrCreate(
+                Arrays.asList(identicalVariantPresentInDbsnp));
```

Comment:
Shouldn't identicalVariantPresentInDbsnp be used in the previous function?

Reply:
what do you mean? in `@Test getOrCreateDbsnpIdenticalVariants()` ? the same variant is used there. here it has "identical" in the name to contrast with the other variable "equivalent"